src/sustainable-web-design.js

```diff
@@ -0,0 +1,58 @@
+"use strict";
+
+/**
+ * Sustainable Web Design
+ *
+ * Newly update calculations and figures from
+ * https://sustainablewebdesign.org/calculating-digital-emissions/
+ *
+ *
+ */
+const { fileSize } = require("./constants");
+const { formatNumber } = require("./helpers");
+
+const KWH_PER_GB = 0.81;
+const GLOABL_INTENSITY = 475;
+const FIRST_TIME_VIEWING_PERCENTAGE = 0.25;
+const RETURNING_VISITOR_PERCENTAGE = 0.75;
+const PERCENTAGE_OF_DATA_LOADED_ON_SUBSEQUENT_LOAD = 0.02;
+
+class SustainableWebDesign {
+  constructor(options) {
+    this.options = options;
+  }
+
+  energyPerVisit(bytes) {
+    const transferedBytesToGb = bytes / fileSize.GIGABYTE;
+    return (
+      transferedBytesToGb * KWH_PER_GB * RETURNING_VISITOR_PERCENTAGE +
```

Comment:
Given this is representing two separate things, why not make this more explicit with named variables for two separate things?

I can't speak for others, but I think it makes it easier to see errors being introduced when during implementation.

I think this is intended to be roughly equivalent to _Energy per visit in kWh (E)_ part of the model from [the SWD page](https://sustainablewebdesign.org/calculating-digital-emissions/). I've broken it onto separate lines to make it a tad easier to read:

> (Data Transfer per Visit (new visitors) in GB x 0.81kWh/GB x 0.75) 
> + 
> (Data Transfer per Visit (returning visitors) in GB x 0.81kWh/GB x 0.25 x 0.02)
> 
> 

The code in this PR here has `FIRST_TIME_VIEWING_PERCENTAGE` and `RETURNING_VISITOR_PERCENTAGE` the other way around when compared to the initial model.

I also missed it the first time when reviewing this PR 😅